src/apps/loan-list/list/list.tsx

```diff
@@ -77,6 +80,7 @@ const List: FC<ListProps> = ({
                 className={`dpl-icon-button ${
                   view === "list" ? "dpl-icon-button--selected" : ""
                 }`}
+                id="test-list"
```

Comment:
Is it intentional to call it "test-list"?
If you want to have a selector for testing it is generally a good practise to use data attributes for marking something to be tested:
https://docs.cypress.io/guides/references/best-practices#Selecting-Elements